scripts/check_doc_ipfs_hashes.py

```diff
@@ -49,6 +49,9 @@
 PACKAGE_TABLE_REGEX = rf"\| {PACKAGE_TYPE_REGEX}\/{VENDOR_REGEX}\/{PACKAGE_REGEX}\/{VERSION_REGEX}(\s|\|)*(?P<hash>{IPFS_HASH_REGEX})\s*\|"
 
 ROOT_DIR = Path(__file__).parent.parent
+# We need to skip the hash for `valory/oracle_hardhat:0.1.0:bafybeie553shfmnds6v7defynjv5kmjkqf2aygj345jbbcssevtnkbodbe`
+# because the demo is not in this repo, but the docs are
+HASH_SKIPS = ("bafybeie553shfmnds6v7defynjv5kmjkqf2aygj345jbbcssevtnkbodbe",)
```

Comment:
This is not ideal. Eventually, we will end up having an outdated hash. I suggest that we open an issue to resolve this. Reading the hash from the `price-estimation` repo and replacing it in the doc file should be enough.
@dvilelaf @jmoreira-valory

Reply:
No I think this needs to be a manual process for now. We can add in the release script that the hash be updated here (and in the docs)! It's like updating a library.

Reply:
Added in the process on f225f937863f9fc2821d663ecb33d90c4edc0c38.